sqlglot/parser.py

```diff
@@ -527,6 +527,7 @@ class Parser(metaclass=_Parser):
         TokenType.PERCENT,
         TokenType.PIVOT,
         TokenType.PRAGMA,
+        TokenType.PUT,
```

Comment:
Let's add this at the end of `identity.sql`:

```sql
SELECT 1 put
```

This tests that the `PUT` token can still be used as, e.g., an identifier.

Reply:
Done, added.